src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,31 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  width: getWidth($cellWidth);
+  margin: auto;
+  padding: $padding;
+
+  $days:
+    "mon" 1,
+    "tue" 2,
+    "wed" 3,
+    "thu" 4,
+    "fri" 5,
+    "sat" 6,
+    "sun" 7;
+
+  @each $name, $value in $days {
+    &--start-day-#{$name}:first-child {
+      margin-left: ($value - 1) * ($cellWidth + 1px);
+    }
+  }
+
+  @for $var from 28 through 31 {
```

Comment:
There needs to be one more for loop to hide all days after 29
And just like before there needs to be a space before the ```scss :first-child``` for the same reason with the above comment.


```scss
  @for $var from 28 through 31 {
    @for $i from 1 through 31 - $var { // This code goes through 1,2,3... hiding the elements after the 29th day
      &--month-length-#{$var} :nth-child(#{$var + $i}) {  // day 29 is an example I'm using.. Works for 28,29,30
        display: none;
      }
    }
  }
```

Reply:
> There needs to be one more for loop inside to hide all days after 29 And just like before there needs to be a space before the `:first-child` for the same reason with the above comment.
> 
> ```css-scss
>   @for $var from 28 through 31 {
>     @for $i from 1 through 31 - $var { // This code goes through 1,2,3... hiding the elements after the 29th day
>       &--month-length-#{$var} :nth-child(#{$var + $i}) {  // day 29 is an example I'm using.. Works for 28,29,30
>         display: none;
>       }
>     }
>   }
> ```

Thanks for your comment. Just I don't get why shall I add the second loop while the first one does the job?